test/dag_connection_test.py

```diff
@@ -32,30 +30,12 @@ def setup_network(self):
     def run_connect_dag_scenario(self):
         self.log.info("Running connect_dag scenario")
         self.warcli("scenarios run-file test/framework_tests/connect_dag.py")
-
-        start_time = time.time()
-        while time.time() - start_time < self.scenario_timeout:
-            running_scenarios = self.rpc("scenarios_list_running")
-            if not running_scenarios:
-                self.log.info("Scenario completed successfully")
-                return
-
-            if len(running_scenarios) == 1 and not running_scenarios[0]["active"]:
-                self.log.info("Scenario completed successfully")
-                return
-
-            time.sleep(1)
-
-        self.log.error(f"Scenario did not complete within {self.scenario_timeout} seconds")
-        self.stop_running_scenario()
-        raise AssertionError(f"Scenario timed out after {self.scenario_timeout} seconds")
-
-    def stop_running_scenario(self):
-        running_scenarios = self.rpc("scenarios_list_running")
-        if running_scenarios:
-            pid = running_scenarios[0]["pid"]
-            self.log.warning(f"Stopping scenario with PID {pid}")
-            self.warcli(f"scenarios stop {pid}", False)
+        self.log_expected_msgs = [
+            "Successfully ran the connect_dag.py scenario using a temporary file"
```

Comment:
Of course, now that https://github.com/bitcoin-dev-project/warnet/pull/406 is merged, we have an easier way to do this :-P

Reply:
True, but I think it will be helpful to be able to assert for specific things in our logs